tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -292,4 +292,37 @@ spec:
 			helper.MatchAllInOutput(jsonOut, []string{msg, msgWithQuote})
 		})
 	})
+
+	Context("When deleting Operator backed service", func() {
+		It("should be able to delete a service existing on the cluster and fail otherwise", func() {
```

Comment:
use `odo service delete` in the other tests as well like https://github.com/openshift/odo/pull/3564/files#diff-49f09fb5341717455c0421b3c592e685R283

Reply:
Done.